qutip/tests/test_transfertensor.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import qutip as qt
+import numpy as np
+from qutip.nonmarkov.transfertensor import *
```

Comment:
We should add any more uses of `import *` in QuTiP -- it just makes it hard to see what the code is doing. Just import `ttmsolve`.